.github/workflows/build.yml

```diff
@@ -0,0 +1,92 @@
+name: Build, Test and Deploy
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  SERVER_HOST: http://localhost:3001
+  WEBSOCKET_ENDPOINT: ws://localhost:3001/subscriptions
+  DEFAULT_DOMAIN: localhost:4200
+  BROCCOLI_PERSISTENT_FILTER_CACHE_ROOT: ./broccoli-tmp
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - name: Cancel previous running workflows
+        uses: fkirc/skip-duplicate-actions@master
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v1
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: install dependencies
+        run: yarn
+      - name: Lint commit message
+        uses: wagoid/commitlint-github-action@v2
+
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: Cache gem
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - name: Bundle install
+        run: |
+          bundle config path vendor/bundle
+          gem install bundler -v '1.17.3'
+          bundle install
+      - name: Test
+        timeout-minutes: 30
+        run: |
+          RAILS_ENV=test bundle exec rake test
+          RAILS_ENV=test bundle exec rake db:migrate && bundle exec rspec --color --format doc
```

Comment:
I can't find the `cc-test-reporter` logic to report the coverage.
Do I miss something?

Reply:
It's me I missed it my bad 😅